app/components/Showcases/Barter.jsx

```diff
@@ -554,6 +570,43 @@ export default class Barter extends Component {
         }
     }
 
+    onMemoChanged = (type, index) => e => {
+        const memos = Object.assign({}, this.state.memo);
+        memos[type][index] = {message: e.target.value, shown: true};
+
+        //TODO: add debounce
```

Comment:
please resolve todo (it looked fine without debounce imo)

Reply:
Ok